hack/run-ci-e2e-test.sh

```diff
@@ -86,12 +86,17 @@ fi
 # https://steps.svc.ci.openshift.org/help/ci-operator#release
 OPERATOR_IMAGE=${OPERATOR_IMAGE:-${IMAGE_FORMAT//\/stable:\$\{component\}//stable:windows-machine-config-operator-test}}
 
-# Setup and run the operator
-if ! run_WMCO $OSDK; then
-  # Try to get the WMCO logs if possible
-  get_WMCO_logs
-  cleanup_WMCO $OSDK
-  exit 1
+# OO_INDEX points to the WMCO bundle index image created through https://docs.ci.openshift.org/docs/how-tos/testing-operator-sdk-operators/ workflow in CI.
+# Setup and run the operator only if $OO_INDEX is not set in CI, and the operator deployment is not available.
+OO_INDEX=${OO_INDEX:-}
+if [ -z "$OO_INDEX" ] && ! oc rollout status deployment windows-machine-config-operator -n openshift-windows-machine-config-operator; then
```

Comment:
Need to skip cleanup as well https://github.com/openshift/windows-machine-config-operator/pull/183/files#diff-9c914ef97321839b726df636cf533083ac1893c90dbe13db77382e043d1f1ba4R128

Reply:
the new workflow does not mention anything about handling cleanup, are you suggesting this because our cleanup is using packagemanifest command as well?

Reply:
Yes, but in general if we arent deploying the operator we shouldnt be removing it.

Reply:
Unless we have a specific need to do so